.babelrc

```diff
@@ -1,12 +1,10 @@
 {
   "presets": [
-    "env",
-    "react"
+    "@babel/preset-env",
+    "@babel/preset-react"
   ],
   "plugins": [
-    "syntax-dynamic-import",
-    "transform-object-rest-spread",
-    "transform-class-properties"
+    
```

Comment:
Empty section :smile:

Reply:
I've removed the whole `plugins` property (hope I didn't misunderstand you again), still failed :(

Reply:
Very strange, `babel-present-env` contains `@babel/plugin-proposal-class-properties`, so we don't need it in our plugins, but looks it was not released right now :confused: , so let's return `@babel/plugin-proposal-class-properties` in plugins array

Reply:
Done.